leveldbstore/store.go

```diff
@@ -0,0 +1,48 @@
+package leveldbstore
+
+import (
+	"github.com/eventials/go-tus"
+	"github.com/mitchellh/go-homedir"
+	"github.com/syndtr/goleveldb/leveldb"
+	"path/filepath"
+)
+
+type LeveldbStore struct {
+	db *leveldb.DB
+}
+
+func NewLeveldbStore() (tus.Store, error) {
+	home, err := homedir.Dir()
```

Comment:
It's better, maybe someone one day could place in `/var` folder for example, or on Windows `C:\ProgramData`.